health/checks/checks.go

```diff
@@ -15,10 +17,19 @@ import (
 // if the file exists.
 func FileChecker(f string) health.Checker {
 	return health.CheckFunc(func() error {
-		if _, err := os.Stat(f); err == nil {
+		absoluteFilePath, err := filepath.Abs(f)
+		if err != nil {
+			return fmt.Errorf("get absolute path for %q error. %s", f, err)
```

Comment:
to be more consistent with other errors maybe `failed to get absolute path for %q: %v`